frontend/src/containers/Profile/ProfileDetail/ProfileDetail.js

```diff
@@ -95,7 +95,7 @@ class ProfileDetail extends Component {
                   user={review.user}
                   title={review.title}
                   date={review.date}
-                  numReplies={review.numReplies}
+                  repliesCount={review.repliesCount}
```

Comment:
Please use replyCount, not repliesCount. I remembered it was the exact name.

Reply:
Also, don’t we need likeCount of reviews and collections on Profile Page?

Reply:
Those props are for rendering CollectionCards and ReviewCards. If we need to show the number of likes for each collection for review, we need to modify the props of them.

Reply:
Why? Actually, they already have likeCount prop!

Reply:
Maybe it would be good to use followerCount, not followersCount, too.

Reply:
I don't get the point. I brought the props format from CollectionCard and ReviewCard components, which are on our master branch. Neither of them have the props named 'likeCount' for now. If 'LikeCount props is omitted from both of CollectionCard and ReviewCard component' is what you wanted to say, then I think we need to resolve the problem in another issue.

Reply:
I checked the card components again, so ReviewCard now has different format with what I had been worked. I will resolve it, but still CollectionCard does not have likeCount props. It needs to be modified.

Reply:
@njsh4261  Ok, now I understood why you mentioned CollectionCard especially. Right, it seems to have a problem. However, it would be fixed to have 'props' likeCount. I don't want to insist strongly, but isn't it OK to pass props which don't exist in the component, which will certainly have the props soon? But I'll agree with your judgement.